docs/snapshotting/snapshot-support.md

```diff
@@ -532,8 +532,8 @@ Firecracker.
 to vsock packet loss, leads to Vsock device breakage when doing a snapshot while
 there are active Vsock connections.
 
-   _**Workaround**_: Close all active Vsock connections prior to snapshotting
-   the VM.
+   _**Workaround**_: All active Vsock connections prior to snapshotting
+   the VM are forcibly closed by sending a specific event called VIRTIO_VSOCK_EVENT_TRANSPORT_RESET.
```

Comment:
We need to also include some details for the users to clearly understand how they should build their vsock apps wrt snapshots.